proto/mixer.proto

```diff
@@ -595,17 +595,30 @@ message GetStatValueRequest {
   string place = 1;
   // dcid of the stat var.
   string stat_var = 2;
-  // (optional) date of the stat.
+  // (optional) date of the stat. The latest date will be used if unspecified.
   string date = 3;
 }
 
 message GetStatValueResponse {
   double value = 1;
 }
 
+// Request for GetStatSeries service.
+message GetStatSeriesRequest {
+  // dcid of the place.
+  string place = 1;
+  // dcid of the stat var.
+  string stat_var = 2;
+}
 
-service Mixer {
+// Response for GetStatSeries service.
+message GetStatSeriesResponse {
+  // A map from ISO date to stat value.
+  map<string, double> series = 1;
```

Comment:
Is map preferred to returning a list of pairs sorted by date? That feels a bit more user-friendly, but don't feel strongly.

Reply:
map is a bit easier to access and less unpacking (which we kinda desire for). If user need ordering, then they can get sorted(series) pretty easily. Would lean towards a map here.